pkgs/applications/misc/ArchiSteamFarm/web-ui/default.nix

```diff
@@ -15,20 +13,13 @@ let
     sha256 = "1ajmi2l6xhv3nlnag2kmkblny925irp4gngdc3mniiimw364p826";
   };
 
-in
-nodePackages.package.override {
-  inherit src;
-
-  # upstream isn't tagged, but we are using the latest official commit for that specific asf version (assuming both get updated at the same time)
-  version = ArchiSteamFarm.version;
+  npmDepsHash = "sha256-AY1DFuZkB8tOQd2FzHuNZ31rtLlWujP+3AqsMMB2BhU=";
 
-  nativeBuildInputs = [ pkgs.nodePackages.node-gyp-build ];
+  nativeBuildInputs = [ nodePackages.node-gyp-build nodePackages.webpack-cli ];
```

Comment:
I'm surprised passing these packages is required at all.

Reply:
You're right, fixed!